acceptance/framework/cli/helpers.go

```diff
@@ -0,0 +1,25 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TranslateListOutput takes the raw output from the proxy list command and
+// translates the table into a map.
+func TranslateListOutput(raw []byte) map[string]string {
```

Comment:
I think this can live in the same file as the connect test for now as it is the only place that it is used. Also, given it is a test fixture, it should be fine to not test this.